src/plugins/intel_gpu/src/graph/pooling.cpp

```diff
@@ -224,15 +222,7 @@ std::vector<layout> pooling_inst::calc_output_layouts(pooling_node const& /*node
     if (auto_pad == ov::op::PadType::SAME_UPPER || auto_pad == ov::op::PadType::SAME_LOWER) {
         pads_begin.clear();
         pads_end.clear();
-        OPENVINO_SUPPRESS_DEPRECATED_START
-        ngraph::try_apply_auto_padding(input_shape,
-                                       kernel_size,
-                                       stride,
-                                       dilation,
-                                       auto_pad,
-                                       pads_end,
-                                       pads_begin);
-        OPENVINO_SUPPRESS_DEPRECATED_END
+        ov::util::try_apply_auto_padding(input_shape, kernel_size, stride, dilation, auto_pad, pads_end, pads_begin);
```

Comment:
the function will be removed in https://github.com/openvinotoolkit/openvino/pull/22036

Reply:
Merged